packages/webapp/pages/onboarding.tsx

```diff
@@ -159,6 +161,19 @@ export function OnboardPage(): ReactElement {
   };
 
   const onClickCreateFeed = () => {
+    if (activeScreen === OnboardingStep.EditTag) {
+      const onboardingContentType = getFeatureValue(
+        feature.onboardingContentType,
+      );
+
+      if (onboardingContentType) {
+        shouldEnrollContentType.current = true;
+        return onClickNext();
+      }
+    } else if (activeScreen === OnboardingStep.ContentTypes) {
+      shouldEnrollContentType.current = false;
+    }
```

Comment:
I think you might want this inside line 132 right?
This could be some extreme edge-case where it re-renders and sets to false without ever showing it.
Think it's safer to do it when it shows to reset.

Reply:
removed the move to false, since we don't really need it.